scripts/deploy.sh

```
@@ -1,15 +1,41 @@
 #!/bin/bash
 
+cd /home/ubuntu
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)